_includes/documentation/v3/mediasoup/api/Router.md

```diff
@@ -294,7 +294,7 @@ Creates a new active speaker observer.
 
 Argument    | Type    | Description | Required | Default
 ----------- | ------- | ----------- | -------- | ----------
-`options`   | [ActiveSpeakerObserverOptions](#ActiveSpeakerObserverOptions) | Options. | Yes |
+`options`   | [ActiveSpeakerObserverOptions](#ActiveSpeakerObserverOptions) | Options. | No |
```

Comment:
Good, I've also fix it in `AudioLevelObserver`: https://github.com/versatica/mediasoup-website/commit/8b558b485501a778f932a4f74216f1015ccbf950